generated/index.md

```diff
@@ -11828,8 +14648,17 @@
 * [2021-03-01](web/microsoft.web/2021-03-01/types.md#resource-microsoftwebsites2021-03-01)
 
 ### microsoft.web/sites/backups
-* [2015-08-01](web/microsoft.web/2015-08-01/types.md#resource-microsoftwebsitesbackups2015-08-01)
-* [2016-08-01](web/microsoft.web/2016-08-01/types.md#resource-microsoftwebsitesbackups2016-08-01)
```

Comment:
Same situation as described above for `Microsoft.Maintenance/applyUpdates` resource. This resource defines a [GET by variable name operation](https://github.com/Azure/azure-rest-api-specs/blob/98e17cdd0a433198aa964963f5178ce2f0d8f2b4/specification/web/resource-manager/Microsoft.Web/stable/2016-08-01/WebApps.json#L485-L568) and a [PUT by constant name operation](https://github.com/Azure/azure-rest-api-specs/blob/98e17cdd0a433198aa964963f5178ce2f0d8f2b4/specification/web/resource-manager/Microsoft.Web/stable/2016-08-01/WebApps.json#L440-L484).